src/elasticodbc/es_parse_result.cpp

```diff
@@ -331,7 +331,7 @@ bool AssignRowData(const json_arr_it &row, const schema_type &row_schema,
     size_t i = 0;
     for (auto row_column = row.value_begin(); i < row_schema.size();
          ++row_column, ++i) {
-        // TODO: Milestone-3 use row_schema[i].second's type (or something else)
+        // Use row_schema[i].second's type (or something else)
```

Comment:
Either remove the whole comment, or make a new issue for it and keep the TODO

Reply:
removed